pkg/controllers/scheduling/clustersetbinding_event_handler.go

```diff
@@ -0,0 +1,68 @@
+package scheduling
+
+import (
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/klog/v2"
+
+	clusterlisterv1alpha1 "github.com/open-cluster-management/api/client/cluster/listers/cluster/v1alpha1"
+	clusterapiv1alpha1 "github.com/open-cluster-management/api/cluster/v1alpha1"
+)
+
+type clusterSetBindingEventHandler struct {
+	clusterSetLister     clusterlisterv1alpha1.ManagedClusterSetLister
+	placementLister      clusterlisterv1alpha1.PlacementLister
+	enqueuePlacementFunc enqueuePlacementFunc
+}
+
+func (h *clusterSetBindingEventHandler) OnAdd(obj interface{}) {
+	h.onChange(obj)
+}
+
+func (h *clusterSetBindingEventHandler) OnUpdate(oldObj, newObj interface{}) {
+	h.onChange(newObj)
+}
+
+func (h *clusterSetBindingEventHandler) OnDelete(obj interface{}) {
+	h.onChange(obj)
+}
+
+func (h *clusterSetBindingEventHandler) onChange(obj interface{}) {
+	// skip resource other than ManagedClusterSetBinding
+	clusterSetBinding, ok := obj.(*clusterapiv1alpha1.ManagedClusterSetBinding)
```

Comment:
if we  only care about name/namespace of the binding. We may change it to runtime.Object and get name/namespace by accessor. We also need to handle the  delete case separately.

Reply:
Done.